netlogo-gui/src/main/app/NotificationBanner.scala

```diff
@@ -5,63 +5,74 @@ package org.nlogo.app
 import com.vladsch.flexmark.html.HtmlRenderer
 import com.vladsch.flexmark.parser.Parser
 
-import java.awt.{BorderLayout, Dimension, Graphics}
-import java.awt.event.{ActionEvent, ActionListener, MouseAdapter, MouseEvent}
+import java.awt.{ BorderLayout, Dimension, Graphics, GridBagConstraints, GridBagLayout, Insets }
+import java.awt.event.{ MouseAdapter, MouseEvent }
 import java.util.prefs.Preferences
-import javax.swing.{JButton, JEditorPane, JLabel, JOptionPane, JPanel, JScrollPane, SwingConstants}
-
+import javax.swing.{ JEditorPane, JLabel, JPanel, JScrollPane, SwingConstants }
 import org.nlogo.app.infotab.InfoFormatter
 import org.nlogo.core.I18N
-import org.nlogo.swing.{HoverDecoration, OptionDialog}
-import org.nlogo.theme.{InterfaceColors, ThemeSync}
-
+import org.nlogo.swing.{ CloseButton, CustomOptionPane, HoverDecoration, OptionPane }
+import org.nlogo.theme.{ InterfaceColors, ThemeSync }
 import org.json.simple.parser.JSONParser
-import org.json.simple.{JSONArray, JSONObject}
+import org.json.simple.{ JSONArray, JSONObject }
 
 import scala.io.Source
 
-
 case class JsonObject(eventId: Int, date: String, title: String, fullText: String)
 
-class NotificationBanner() extends JPanel with ThemeSync with HoverDecoration {
-  private var jsonObjectList: List[JsonObject] = List() // Initialize here first
-  private val jsonUrl = "https://ccl.northwestern.edu/netlogo/announce-test.json"
-  // Fetch and populate jsonObjectList in the constructor
-  jsonObjectList = parseJsonToList(fetchJsonFromUrl())
-  private var scrollPane = new JScrollPane()
+class NotificationBanner extends JPanel with ThemeSync with HoverDecoration {
+  private var JsonObjectList: Seq[JsonObject] = List()
+  private val JsonUrl = "https://ccl.northwestern.edu/netlogo/announce-test.json"
 
-  private val initialMessage = getJsonObjectHead.getOrElse("")
-  lazy val lastSeenEventIdKey: String = "lastSeenEventId" // The key for the most recently seen event-id
-  private var editorPane: JEditorPane = new JEditorPane()
+  JsonObjectList = parseJsonToSeq(fetchJsonFromUrl())
+  private var ScrollPane = new JScrollPane()
```

Comment:
There is now a class that correctly synchronizes scroll panes with the color theme, it's at `org.nlogo.swing.ScrollPane`. If I added it after you last merged with laf-widget, don't worry about it now, I'll fix it after you merge this branch.